doc/news/changes/major/20190425RolandRichter

```diff
@@ -0,0 +1,3 @@
+New: The tutorial examples now all use an interpolation formula with a degree depending on the degree of the finite element, which can be changed dynamically.
```

Comment:
```suggestion
New: The tutorial examples now all use a quadrature formula with a degree depending on the degree of the
finite element, so that one changing the finite element degree does not also require changing by hand the number
of quadrature points.
```